argus/backends/tempest/tempest_backend.py

```diff
@@ -58,6 +57,7 @@ class BaseTempestBackend(base_backend.CloudBackend):
     """
     def __init__(self, name, userdata, metadata, availability_zone):
         if userdata:
+            # pylint: disable=deprecated-method
             userdata = base64.encodestring(userdata)
```

Comment:
With the above annotation, it will not. Any suggestion in order to avoid pylint-disable ?